docs/docs/en/development/development-environment-setup.md

```diff
@@ -2,12 +2,11 @@
 
 ## Software Requests
 
-Before setting up the DolphinScheduler development environment, please make sure you have installed the software as below:
```

Comment:
I think this description needs to be retained.

Reply:
ok，plus it's more user-friendly